lib/resources/majors.tsx

```diff
@@ -2711,6 +2711,308 @@ const bsMolCell: Major = {
+        {
+          required_credits: 6,
+          description: '<b>Biochem with lab</b>',
+          criteria:
+            '(AS.020.305[C]^AND^AS.020.315[C])^OR^AS.250.253[C]^OR^AS.250.254[C]',
+        },
+        {
+          required_credits: 6,
+          description: '<b>Genetics with lab</b>',
+          criteria: 'AS.020.303[C]^AND^AS.020.340[C]',
+        },
+        {
+          required_credits: 6,
+          description: '<b>Cell Bio with lab</b>',
+          criteria: 'AS.020.306[C]^AND^AS.020.316[C]',
+        },
+        {
+          required_credits: 6,
+          description: '<b>Comparative Physiology with lab</b>',
+          criteria: 'AS.020.374[C]^AND^AS.020.377[C]',
+        },
+      ],
+    },
+    {
+      name: 'Focus Area',
+      required_credits: 9,
+      min_credits_per_course: 1,
+      pathing: true,
+      description:
+        'Nine credits of 300-level or higher approved courses from one of four focus areas: systems neuroscience, ' +
+        'cognitive neuroscience, computational neuroscience, or cellular and molecular neuroscience',
+      criteria:
+        'NEUR-ST[T]^OR^NEUR-CG[T]^OR^NEUR-CP[T]^OR^NEUR-CM[T]^OR^AS.080.3[C]^OR^AS.200.3[C]',
+      fine_requirements: [
+        {
+          required_credits: 9,
+          description: '<b>Systems neuroscience (ST)</b> <br /> ',
+          criteria: 'NEUR-ST[T]',
+        },
+        {
+          required_credits: 9,
+          description: '<b>Cognitive neuroscience (CG)</b> <br /> ',
+          criteria: 'NEUR-CG[T]',
+        },
+        {
+          required_credits: 9,
+          description:
+            '<b>Computational neuroscience (CP)</b> <br /> ' +
+            'EN.553.291 (or AS.110.201 and AS.110.302) is required in addition to the credits required of the focus area.',
+          criteria: 'NEUR-CP[T]',
+        },
+        {
+          required_credits: 9,
+          description:
+            '<b>Cellular and molecular neuroscience (CM)</b> <br /> ' +
+            'AS.020.306 and AS.020.316 must be selected as the required biology course with lab.',
+          criteria: 'NEUR-CM[T]',
+        },
+      ],
+    },
+    {
+      name: 'Elective Course',
+      required_credits: 3,
+      min_credits_per_course: 3,
+      description:
+        'Three credits of 300-level or higher approved course outside of selected focus area',
+      criteria: 'AS.080.3[C]',
+    },
+    {
+      name: 'Research',
+      required_credits: 6,
+      min_credits_per_course: 0.5,
+      description:
+        'Six credits of neuroscience research, obtained through work in one of the neuroscience laboratories ' +
+        'participating in the program. Read more about <a href="https://krieger.jhu.edu/neuroscience/research/research-credit-requirements/">' +
+        'research credit requirements</a>.',
+      criteria: 'AS.080.500[C]^OR^AS.080.541[C]', // TODO: course # for neuro research?
+      fine_requirements: [
+        {
+          required_credits: 0.5,
```

Comment:
At least 2 semesters of this class is required so change `required_credits` to 1